core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -30,7 +30,7 @@ class PerfConfig(args: Array[String]) {
     .withRequiredArg
     .describedAs("size")
     .ofType(classOf[java.lang.Integer])
-    .defaultsTo(5000)
+    .defaultsTo(-1)
```

Comment:
this option is a little strange as the default is overridden dynamically. I've updated the describedAs and added a comment for clarity.